cost-optimization/gke-keda/README.md

```diff
@@ -0,0 +1,7 @@
+# Pub/Sub on Kubernetes Engine
+
+[![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.svg)](https://ssh.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/kubernetes-engine-samples&cloudshell_tutorial=README.md&cloudshell_workspace=gke-keda/)
```

Comment:
Workspace should be `cost-optimization/gke-keda/`

Reply:
Done.